dask_cudf/tests/test_groupby.py

```diff
@@ -15,9 +15,7 @@
         lambda df: df.groupby("x").count(),
         lambda df: df.groupby("x").min(),
         lambda df: df.groupby("x").max(),
-        lambda df: df.groupby("x").std(),
         lambda df: df.groupby("x").y.sum(),
-        lambda df: df.groupby("x").y.std(),
```

Comment:
Hrm, we should still be able to support this in dask-cudf even though cudf doesn't support it.  The parallel algorithm doesn't use `cudf.std` to implement the std of the full dataframe.

Reply:
In the case of `std` I suspect that the failure in with `a`, when we do the reference test.  Instead, we might use pandas rather than cudf here to get the expected value.  We might place these two cases in a separate test that does the following replacement

```diff
-       a = func(gdf).to_pandas()
+       a = func(gdf.to_pandas())
```